pkg/aggregator/internal/tags/store.go

```diff
@@ -106,6 +107,7 @@ func (tc *Store) Insert(key ckey.TagsKey, tagsBuffer *tagset.HashingTagsAccumula
 		entry = &Entry{
 			tags: tagsBuffer.Copy(),
 		}
+		sort.Strings(entry.tags)
```

Comment:
I'm extremely worried to see a sort appear here.
I've put a lot of efforts into not having to sort tags to still be able to create a stable context key. Adding this sort back to the equation is breaking all these efforts which brought huge win in the past.

Maybe this branch isn't called by what I think it is, but still, a sort in such a place (on all inserts calls) isn't looking good. Is this dev/debug code?